src/main/java/org/junit/experimental/theories/ParameterSignature.java

```diff
@@ -34,7 +34,7 @@
         map.put(b, a);
     }
     
-    public static ArrayList<ParameterSignature> signatures(Method method) {
+    public static List<ParameterSignature> signatures(Method method) {
```

Comment:
I personally don't think it's worth it to deprecate the method and replace it with a similarly-named method that returns a `List`